app/controllers/carto/api/user_presenter.rb

```
@@ -37,6 +37,14 @@ def to_poro
         poro
       end
 
+      def to_poro_without_id
```

`to_poro_without_id` feels arbitrary. Maybe changing `to_poro` to delete `id` key if `id` is `nil` is consistent and better. I was even thinking about using `reject` four times instead of creating this method! Feel free to ignore this, though, as javitonino has already CR'd the PR.